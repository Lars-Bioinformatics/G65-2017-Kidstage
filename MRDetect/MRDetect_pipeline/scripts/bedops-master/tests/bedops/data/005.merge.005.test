chr4	5	11
chr4	16	17
chr6	0	5
chr6	120	121
