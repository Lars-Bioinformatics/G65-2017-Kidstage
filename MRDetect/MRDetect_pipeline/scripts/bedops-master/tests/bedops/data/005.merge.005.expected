chr4	5	11
chr4	16	17
chr6	0	11
chr6	16	118
chr6	119	121
chr7	2	102
chr7	105	116
chr7	117	151
